grpc_codegen/avs.proto

```diff
@@ -468,6 +475,49 @@ message UserTriggerTaskResp {
   ExecutionStatus status = 2;
 }
 
+message CreateOrUpdateSecretReq {
```

Comment:
Is this new API interface reflected in the sdk-js source code yet?

We need to make sure all the changes in avs.proto are factored into the source code.